crates/bin/pcli/src/command/init.rs

```diff
@@ -106,8 +110,53 @@ impl SoftKmsInitCmd {
     }
 }
 
+#[derive(Debug, clap::Subcommand)]
+pub enum ThresholdInitCmd {
+    /// Use a centralized dealer to create config files for each signer.
+    Deal {
+        /// The minimum number of signers required to make a signature (>= 2).
+        #[clap(short, long)]
+        threshold: u16,
+        /// One file name for each signer, where the dealt config will be saved.
+        #[clap(short, long, value_delimiter = ' ', multiple_values = true)]
+        out: Vec<Utf8PathBuf>,
+    },
+}
+
+impl ThresholdInitCmd {
+    fn exec(&self, grpc_url: Url) -> Result<()> {
+        use penumbra_custody::threshold;
+
+        match self {
+            ThresholdInitCmd::Deal { threshold: t, out } => {
+                if *t < 2 {
+                    anyhow::bail!("threshold must be >= 2");
+                }
+                let configs = threshold::Config::deal(&mut OsRng, *t, out.len() as u16)?;
+                println!("Writing dealt config files.");
+                for (config, path) in configs.into_iter().zip(out.iter()) {
+                    let full_viewing_key = config.fvk().clone();
+                    let config = PcliConfig {
+                        custody: CustodyConfig::Threshold(config),
+                        full_viewing_key,
+                        grpc_url: grpc_url.clone(),
+                        view_url: None,
+                        disable_warning: false,
+                    };
+                    config.save(path)?;
+                }
+                Ok(())
+            }
+        }
+    }
+}
+
 impl InitCmd {
     pub fn exec(&self, home_dir: impl AsRef<camino::Utf8Path>) -> Result<()> {
+        if let InitSubCmd::Threshold(cmd) = &self.subcmd {
+            cmd.exec(self.grpc_url.clone())?;
+            return Ok(());
```

Comment:
doesn't returning here mean that the `PcliConfig` doesn't get created? I think then later when we try to use `pcli threshold` we'll get an error as it's inspecting the contents of the `PcliConfig` to determine the configured custody backend